server.js

```diff
@@ -21,122 +23,30 @@ db.on('error', (err) => {
   console.log(err);
 });
 
-// PARSERS:
+// Parsers:
 app.use(bodyParser.urlencoded({extended: true}));
-// app.use(express.urlencoded()) - parsing URL-encoded bodies
-app.use(bodyParser.json());
-// the same as express.json() - parsing JSON bodies sent by API clients
-
-// Import Routes
-const login = require('./routes/api/login');
-const registration = require('./routes/api/registration');
-const profile = require('./routes/api/profile');
+app.use(bodyParser.json( {extended: true }));
 
 
-// Routes
+// Routes:
 app.get('/', (req, res) => {
-  return res.send('Hello');
+  return res.status(200).json({status: 'OK!'})
```

Comment:
is it smth like health check ?

Reply:
yeah, not sure, what it has to return in such cases